js/main.js

```diff
@@ -31,3 +31,47 @@ const appendElementWithText = function (parent, tagName, text) {
   element.appendChild(textNode)
   parent.appendChild(element)
 }
+
+const buildRecipeDetails = function (recipeDetails) {
+  const dl = document.createElement('dl')
+
+  for (let i = 0; i < recipeDetails.length; i++) {
+    const { label, value } = recipeDetails[i]
+    appendElementWithText(dl, 'dt', label)
+    appendElementWithText(dl, 'dd', value)
+  }
+  return dl
+}
+
+const displayRecipe = function (recipe) {
+  const div = document.getElementById('display-recipe')
```

Comment:
Is there a better name we could use for this var than `div`? Perhaps a less generic one, that will make it clear what you are adding where later on in the same function?